mitodl/bootcamp-ecommerce
# Added seed data command for setting application state

#### What are the relevant tickets?
No ticket - adds some seed data functionality alongside the command added in #927 

#### What's this PR do?
Adds a management command to set an application to a desired state

#### How should this be manually tested?
Create an application for some user and bootcamp run, then run the management command to set the state - ex:`set_application_state -i <your application id> --state AWAITING_PAYMENT`. Check that the application fields have the expected data, including related objects like submissions. From there you can use different states for the command and confirm accordingly.

#### Screenshots (if appropriate)
![ss 2020-11-20 at 15 52 25 ](https://user-images.githubusercontent.com/14932219/99851133-a32bd600-2b4c-11eb-8729-114aecf79dd8.png)
